build.sbt

```diff
@@ -40,17 +40,16 @@ lazy val server = (project in file("server"))
     scalaVersion := "2.11.7",
     scalaJSProjects := clients,
     pipelineStages := Seq(scalaJSProd, gzip),
-    herokuAppName in Compile := "scala-exercises",
   libraryDependencies ++= Seq(
       filters,
       jdbc,
       evolutions,
       cache,
     ws,
-      "org.scalaexercises" %% "runtime" % "0.0.0-SNAPSHOT" changing(),
-      "org.scalaexercises" %% "content-stdlib" % "0.0.0-SNAPSHOT",
-      "org.scalaexercises" %% "content-cats" % "0.0.0-SNAPSHOT",
-      "org.scalaexercises" %% "content-shapeless" % "0.0.0-SNAPSHOT",
+      "org.scala-exercises" %% "runtime" % "0.0.1" changing(),
```

Comment:
These shall remain snapshots until we have a final version because we want up to date snapshots of master.